CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 
 - Add `EnvelopeStack` and `SQLiteEnvelopeStack` to manage envelopes on disk. ([#3855](https://github.com/getsentry/relay/pull/3855))
 - Add `client_sample_rate` to spans, pulled from the trace context ([#3872](https://github.com/getsentry/relay/pull/3872)).
+- Add new trimming behavior in the derive macros. ([#3877](https://github.com/getsentry/relay/pull/3877))
```

Comment:
```suggestion
- Introduce `trim = "disabled"` type attribute to prevent trimming of spans. ([#3877](https://github.com/getsentry/relay/pull/3877))
```